### PR TITLE
fix go redirect

### PR DESCRIPTION
**Pull request by dprotaso**
we've been getting 404s with netlify with the go get redirect.

I opened an netlify support issue and they suggested that the cause might be because the `_redirect` format isn't valid (`go-get=1`) 

Part of https://github.com/knative/docs/pull/5532

### CODE REVIEW

**Comment by dprotaso** [comment]
Going to close this - netlify support said

> We detected a small bug on the way we handled these requests within the same CDN node. Mind trying to hit the 404's are still seen?

**Comment by dprotaso** [comment]
Actually we should still do some cleanup

File: golang/_redirects [hunk 1]
```diff
@@ -5,75 +5,73 @@
...
-/networking/* go-get=1 /golang/networking.html 200
-/operator/* go-get=1 /golang/operator.html 200
-/pkg/* go-get=1 /golang/pkg.html 200
+/actions/* go-get=:go-get /golang/actions.html 200
+/async-component/* go-get=:go-get /golang/async-component.html 200
+/caching/* go-get=:go-get /golang/caching.html 200
+/client/* go-get=:go-get /golang/client.html 200
+/client-pkg/* go-get=:go-get /golang/client-pkg.html 200
+/community/* go-get=:go-get /golang/community.html 200
+/container-freezer/* go-get=:go-get /golang/container-freezer.html 200
```
Comment (psschwei) [inline golang/_redirects:14]: should we delete this one since we're archiving the repo whenever I remember to actually do it?

Comment (dprotaso) [reply golang/_redirects:14]: When you archive the repo - we can regenerate this config that will remove it.

**Review by psschwei — APPROVED** [review]
/lgtm

**Pull request merged**